frontend/lib/src/enums/transaction_enum.dart

```diff
@@ -0,0 +1,59 @@
+// ignore_for_file: constant_identifier_names
+
+enum TransactionTypes { CREDIT, DEPT, TRANSFER }
+
+enum Category {
```

Comment:
category for what

Reply:
Sir Good afternoon, this category was from `/models/transaction.dart/` I just transfer it here in `/src/enums/transaction_enum.dart` and I believe this enum was used somewhere. so I didn't  change the name.

Reply:
`Category` is ambiguous, change enum name